actionpack/lib/action_dispatch/routing/route_set.rb

```diff
@@ -479,7 +480,7 @@ def #{name}
       def url_helpers(supports_path = true)
         routes = self
 
-        Module.new do
+        @url_helpers_cache[!!supports_path] ||= Module.new do
```

Comment:
Should we use a hash or just define two separated instance variables like it was when we used to have this cache?

Reply:
👍 I switched to the previous structure and method naming.